core/src/main/java/hudson/PluginManager.java

```diff
@@ -1634,7 +1634,7 @@ public HttpResponse doProxyConfigure(StaplerRequest req) throws IOException, Ser
     @RequirePOST
     public HttpResponse doUploadPlugin(StaplerRequest req) throws IOException, ServletException {
         try {
-            Jenkins.get().checkPermission(UPLOAD_PLUGINS);
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
Strictly speaking, none of the permission checks in this file matter, but if System Read is about to land, it probably makes sense to keep them.